src/main/java/ro/unibuc/link/data/CollectionEntity.java

```diff
@@ -0,0 +1,20 @@
+package ro.unibuc.link.data;
+
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+
+import java.util.List;
+
+@Data
+public class CollectionEntity {
+    @Id
+    private String collectionName;
+    private List<UrlEntity> urls;
```

Comment:
Please make special entity to manage this particular data as they do not require deleteWord and internalUrl.
An entity with a name and external url would suffice.